decidim-core/app/cells/decidim/user_activity/show.erb

```diff
@@ -1,32 +1,24 @@
-<div class="user-activity">
-  <div id="activities" class="columns small-12">
-    <% if activities.length == 0 %>
-      <div class="callout warning">
-        <%= t "decidim.user_activity.index.no_activities_warning" %>
-      </div>
-    <% else %>
-      <div class="row">
-        <div class="show-for-mediumlarge filters__left-legend">
-          <%= filter_form_for context[:filter] do |form| %>
-            <div class="columns mediumlarge-6 large-5">
-              <%= form.custom_select :resources,
-                                     :resource_type,
-                                     resource_types,
-                                     legend_title: t("title", scope:"decidim.components.component_order_selector"),
-                                     label: false,
-                                     selected: context[:filter].resource_type,
-                                     include_blank: t("all_types", scope: "decidim.components.component_order_selector.order") %>
-            </div>
-          <% end %>
-        </div>
-      </div>
-      <div class="row" id="activities-container">
-        <%= cell "decidim/activities", activities %>
-      </div>
-
-      <div class="row" id="decidim-paginate-container">
-        <%= decidim_paginate activities %>
-      </div>
+<div class="profile__activity">
+    <%= redesigned_filter_form_for context[:filter], url_for, class: "new_filter profile__activity__filter-container" do |form| %>
+      <%# REDESIGN_PENDING: allow passing a default element checked %>
+      <%= form.collection_radio_buttons(
```

Comment:
![Profile activity filters](https://user-images.githubusercontent.com/864340/183032885-3fab9ec1-b1ea-4f07-8028-ff569c4c0e0a.png)

This is much better than in my previous review but I still feel that showing **only** the icons for mobile screens is confusing as this is not something that the user uses every day. Icons are OK by themselves if they are easy to recognize by the user and they are generally familiar with them by using them very often.

I think a dropdown element would be better on mobile in this situation as you can only select one at a time. The dropdown could also show the icon with the label for that element.